docs/language/crypto.mdx

```diff
@@ -107,11 +127,23 @@ struct PublicKey {
     ): Bool
 
     /// Verifies the proof of possession of the private key.
-    /// This function is only implemented if the signature algorithm of the public key is BLS, it errors if called with any other signature algorithm. 
+    /// This function is only implemented if the signature algorithm of the public key is BLS (BLS_BLS12_381).
+    /// It errors if called with any other signature algorithm. 
     pub fun verifyPoP(_ proof: [UInt8]): Bool
 }
 ```
 
+<Callout type="info">
+
+🚧 Status: VerifyPoP is going to be available in an upcoming release of Cadence
+
+</Callout>
```

Comment:
This should be released now, correct?

```suggestion
```

Reply:
Yes, I thought this PR will get merged earlier, and then I can remove these callouts after the spork :) 
I'll remove them now.